cdmbase/fullstack-pro
# redux first history

#342 
With redux-first-history, I met some problems;
- First, remove connected-react-router package from node modules.
- I could not use react-router components such as Route, Link outside of frontend-server.

To test this branch please follow;
`yarn build:clean`
`yarn install`
`lerna exec --scope='@sample-stack/counter-module-browser' yarn build`
`yarn start`

Please check @cdmbase
